example/patches/react-native+0.77.0.patch

```diff
@@ -0,0 +1,114 @@
+diff --git a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/defaults/DefaultReactHost.kt b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/defaults/DefaultReactHost.kt
+index f0d0cdc..89fcfa7 100644
+--- a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/defaults/DefaultReactHost.kt
++++ b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/defaults/DefaultReactHost.kt
+@@ -16,7 +16,7 @@ import com.facebook.react.bridge.ReactContext
+ import com.facebook.react.common.annotations.UnstableReactNativeAPI
+ import com.facebook.react.common.build.ReactBuildConfig
+ import com.facebook.react.fabric.ComponentFactory
+-import com.facebook.react.runtime.JSCInstance
++import com.facebook.react.runtime.JSRuntimeFactory
+ import com.facebook.react.runtime.ReactHostImpl
+ import com.facebook.react.runtime.cxxreactpackage.CxxReactPackage
+ import com.facebook.react.runtime.hermes.HermesInstance
+@@ -58,7 +58,7 @@ public object DefaultReactHost {
+       jsMainModulePath: String = "index",
+       jsBundleAssetPath: String = "index",
+       jsBundleFilePath: String? = null,
+-      isHermesEnabled: Boolean = true,
++      jsRuntimeFactory: JSRuntimeFactory? = null,
```

Comment:
See my comments on the PR upstream as we can't ship this change as it is, so we'll have to update the patch here as well.

We can merge it for the time being though

Reply:
updated patches and will merge this pr until the nightlies include the changes we need